packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -260,8 +260,13 @@ export async function deployArmTemplates(ctx: SolutionContext): Promise<Result<v
 }
 
 async function getParameterJson(ctx: SolutionContext) {
+  if (!ctx.targetEnvName) {
```

Comment:
If `ctx.targetEnvName` not set, should we use `default` as the default value?

Reply:
I would prefer to manage the default value in one place, otherwise the fallback login will be hard to maintain.